crates/matrix-sdk/src/config/request.rs

```diff
@@ -98,7 +98,7 @@ impl RequestConfig {
         self
     }
 
-    /// The number of times a request should be retried. The default is no limit
+    /// The number of times a request should be retried. The default is 0 to disable retries.
```

Comment:
I think this change is wrong: the default (without calling the function) is `None`, which means no limit. 0 means no retries, but they're different things

Reply:
But what does "no limit" mean? It sound like it would retry forever.

Reply:
Yes, that's exactly what it means.